app/main.py

```diff
@@ -1,3 +1,66 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float | int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        if isinstance(self.km, float):
+            return f"Distance: {self.km} kilometers."
+        elif isinstance(self.km, int):
+            return f"Distance: {self.km} kilometers."
```

Comment:
The `else` statement here is unnecessary. You can directly return the string without using `elif` since the previous condition already returns a value if true. This is suggested in the checklist to make the code cleaner.